lib/plugins/keys.js

```diff
@@ -0,0 +1,113 @@
+import { isInteractiveElement, isModifierUsed } from '../utils';
+
+export default shower => {
+    const { container, player } = shower;
+    const containerElement = container.getElement();
+    containerElement.addEventListener('keydown', event => {
+        if (event.defaultPrevented) return;
+        if (!shower.isHotkeysEnabled()) return;
```

Comment:
I can imagine 2 use cases for having `isHotkeysEnabled`, `enableHotkeys`, and `disableHotkeys` methods:

1. End user wants deliberately turn them off. However, he won't be able to switch slides (on desktop) or quit `full` mode (except for `#hash` change).
2. Plugin author has some stuff on a slide that uses same bindings. In that case, they should call `event.preventDefault()`. Otherwise, it is risky to expose such functionality.

With that in mind, I would rather remove those methods. However, I think I am missing something important.

Reply:
I don’t think there‘s a strong use case for disabling shortcuts. So I’d remove those methods for now, until there’s one.